configs/nncf_compression/vehicle-detection/vehicle-detection-0200_int8.py

```diff
@@ -0,0 +1,185 @@
+# model settings
```

Comment:
should not it be in training_extensions?

Reply:
Yes, it should!

Reply:
Fixed